### PR TITLE
Add error logging to project

### PR DESCRIPTION
**Pull request by fredlawl**
It is difficult to debug an application that does not produce any erorr
logging. Add error logging to project through an error file and
console output.

Signed-off-by: Frederick Lawler <fred@fredlawl.com>

### CODE REVIEW

File: tagging_tracker/settings.py [hunk 1]
```diff
@@ -163,3 +163,24 @@
...
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'handlers': {
+        'console': {
+            'class': 'logging.StreamHandler'
+        },
+        'file': {
+            'class': 'logging.FileHandler',
+            'filename': '/logs/error.log'
```
Comment (kolkheang) [inline tagging_tracker/settings.py]: Does this need to have the prefix `/code/` path?

File: tagging_tracker/settings.py [hunk 2]
```diff
@@ -163,3 +163,24 @@
...
+        }
+    },
+    'loggers': {
+        '': {  # 'catch all' loggers by referencing it with the empty string
+            'handlers': ['console', 'file'],
+            'level': 'DEBUG',
```
Comment (kolkheang) [inline tagging_tracker/settings.py]: Can you use the variable debug above?

Comment (kolkheang) [reply tagging_tracker/settings.py]: if DEBUG is true, set to 'DEBUG' else, set to info?
https://github.com/codeforkansascity/tagging_tracker_backend/pull/43/files#diff-5fbc47b4c304f27a99656b4293e9c029R23

Comment (fredlawl) [reply tagging_tracker/settings.py]: I think that when we launch this, we would want the error logging on the server so we can have some insight if something goes wrong. If you want me to change that, I can.

**Pull request merged**